clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -795,10 +815,11 @@ def merge_philips_diffusion(
     multipart_id = _get_multipart_id(
         PhilipsNumberOfParts.from_int(int(number_of_parts)), run_num
     )
-    if multipart_id is not None:
-        data = json.loads(json_file.read_text())
-        data["MultipartID"] = multipart_id
-        json.dump(data, json_file, indent=4)
+    with open(json_file, "r+") as f:
+        if multipart_id is not None:
+            data = json.load(f)
```

Comment:
Why these changes ?

Reply:
I don't know why, but this doesn't work. So I changed back to what I had done in the first place, which I does !
It may not be optimal though, so we can look at it together

Reply:
OK, we'll take a look at the error you're getting this afternoon.